gobblin-compaction/src/main/java/org/apache/gobblin/compaction/verify/InputRecordCountHelper.java

```diff
@@ -71,6 +75,7 @@ public InputRecordCountHelper(State state) {
     try {
       this.fs = getSourceFileSystem (state);
       this.state = state;
+      this.extensionName = state.getProp(COMPACTION_OUTPUT_EXTENSION, DEFAULT_COMPACTION_OUTPUT_EXTENSION);
```

Comment:
looks like it supports a list of extension as in line 97. How about using a new config key that indicates it supports a extensions list? It also helps decouple the configs between `InputRecordCountHelper` and `CompactorOutputCommitter`.

Reply:
I think making this configurable in the first place is a mistake - the extension should always stay consistent with the extension that you specified as part of compaction job. I get the point of decouple the configs, but they are needed in different places and that's why it is public static. ( If strict decoupling is applied then anything should be private or package-private static, which is not realistic